pkg/gator/errors.go

```diff
@@ -18,6 +18,10 @@ var (
 	// ErrNotASyncSet indicates the user-indicated file does not contain a
```

Comment:
Can you pls also update this comment?
```suggestion
	// ErrNotAGVKManifest indicates the user-indicated file does not contain a GVK Manifest
```

Reply:
done